src/styles/colors.tsx

```diff
@@ -24,6 +24,7 @@ enum Colors {
   error = '#EA6042',
   errorDark = '#C93717',
   errorLight = '#FBF2F0',
+  periwinkle = '#869AEF',
```

Comment:
is this used anywhere?

Reply:
whoopps, that was from me trying to make one of the icons work.